six/modules/python/six.sicd/tests/test_slant_to_output.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import os
+import sys
+
+import numpy
+
+import coda.sio_lite as sio_lite
+from coda.coda_types import RowColDouble, RowColSizeT, VectorString
+from coda.math_poly import Poly2D
+import pysix.six_sicd as six_sicd
+from pysix.six_sicd import AreaPlaneUtility, SixSicdUtilities
+
+
+def getSchemaPaths(pathname):
+    schemaPaths = VectorString()
+    if pathname:
+        schemaPaths.push_back(pathname)
+    elif 'SIX_SCHEMA_PATH' in os.environ:
+        schemaPaths.push_back(os.environ['SIX_SCHEMA_PATH'])
+    else:
+        schemaPaths.push_back(six_sicd.schema_path())
+    return schemaPaths
+
+
+def findOutputToSlantPolynomials(complexData, orderX, orderY):
+    polynomialFitter = SixSicdUtilities.getPolynomialFitter(complexData)
+    sampleSpacing = RowColDouble(complexData.grid.row.sampleSpacing,
+                                 complexData.grid.col.sampleSpacing)
+    offset = RowColSizeT(complexData.imageData.firstRow,
+                         complexData.imageData.firstCol)
+    scpPixel = RowColDouble(complexData.imageData.scpPixel.row,
+                            complexData.imageData.scpPixel.col)
+    toSlantRow = Poly2D()
+    toSlantCol = Poly2D()
+    polynomialFitter.fitOutputToSlantPolynomials(
```

Comment:
This looks good.  One thing... can you do an `%extend` or whatever it is to add an overloading to the `ProjectionPolynomialFitter` Python bindings where it returns these two polynomials rather than taking them in by reference so that it looks more natural in Python?

Reply:
Done.